cmd/influxd/server_integration_test.go

```diff
@@ -1320,9 +1324,27 @@ func Test3NodeServer(t *testing.T) {
 
 	nodes := createCombinedNodeCluster(t, testName, dir, 3, 8190, nil)
 
-	runTestsData(t, testName, nodes, "mydb", "myrp")
-	runTest_rawDataReturnsInOrder(t, testName, nodes, "mydb", "myrp")
+	runTestsData(t, testName, nodes, "mydb", "myrp", 3)
+	runTest_rawDataReturnsInOrder(t, testName, nodes, "mydb", "myrp", 3)
+
+}
+
+// ensure that all queries work if there are more nodes in a cluster than the replication factor
+func Test3NodeClusterPartiallyReplicated(t *testing.T) {
+	t.Skip("...")
```

Comment:
To be clear, this test is being skipping so some stuff it not being tested. Is this what you want?